doc/SETUP_DEVELOPMENT_DOCKER.md

```diff
@@ -28,19 +28,20 @@ afterwards.
 
 ## Setup
 
-Then setup foodsoft development (this will take some time, containers needs
-to be pulled from docker registry and a lot dependencies needs to be installed)
+Then start the databasee server and setup foodsoft development (this will take
+some time, containers needs to be pulled from docker registry and a lot
+dependencies needs to be installed)
 
+    docker-compose -f docker-compose-dev.yml up -d mariadb
     docker-compose -f docker-compose-dev.yml run --rm foodsoft \
-      bundle exec rake foodsoft:setup_development
-
-Do not enable mailcatcher, because this is already included as a docker image.
+      bundle install
+    docker-compose -f docker-compose-dev.yml run --rm foodsoft \
+      bundle exec rake foodsoft:setup_development_docker
 
-Then create an initial database (here seeded with `small.en`) by running
+Optionally an initial database (here seeded with `small.en`) can be loaded by running
 
-    docker-compose -f docker-compose-dev.yml up -d mariadb && sleep 15
     docker-compose -f docker-compose-dev.yml run --rm foodsoft \
-      bundle exec rake db:create db:schema:load db:seed:small.en
+      bundle exec rake db:schema:load db:seed:small.en
```

Comment:
Let's be consistent in whether to use `\` or not (here and elsewhere in the file).

Reply:
IMHO it's consistent. It's only missing when there are multiple commands (vs. multiline commands)

Reply:
Yes, I see now that later invocations with `docker-compose-dev` don't use it - which is good.